src/ex1/script.js

```diff
@@ -0,0 +1,122 @@
+const addBtn = document.querySelectorAll('.add-task-btn');
+const clearAllBtn = document.querySelectorAll('.clear-all-btn');
+const addTaskField = document.querySelectorAll('.add-task-field');
+
+
+const taskList = [];
+addChillMsg();
+updateFooter();
+
+
+
+addBtn[0].addEventListener('click', () => {
+    addTaskFunc();
+});
+
+addTaskField[0].addEventListener("keydown", (event) => {
+    if (event.key === "Enter") {
+        addTaskFunc();
+    }
+});
+
+clearAllBtn[0].addEventListener('click', () => {
+    const tasks = document.querySelectorAll('.tasks');
+    tasks[0].replaceChildren();
+    while (taskList.length) taskList.pop();
+    tasks[0].append(ChillMsg());
+    updateFooter();
+});
+
+function addTaskFunc() {
+    const newTaskField = document.querySelectorAll('.add-task-field');
+    const taskText = newTaskField[0].value;
+    const tasks = document.querySelectorAll('.tasks');
+    console.log(taskText);
+    if (taskText == "")
+        alert("Enter new task!");
+    else if (taskList.indexOf(taskText) !== -1) {
+        alert("Task already exist!\nEnter new task!")
+    }
+    else {
+        if (taskList.length === 0) tasks[0].replaceChildren();
```

Comment:
in terms of readability its not that clear that this line is responsible for clearing the chill message, I would either export it to a function or mention that in a comment